scio-google-cloud-platform/src/main/scala/com/spotify/scio/bigquery/BigQueryTypes.scala

```diff
@@ -291,6 +294,15 @@ case class TimePartitioning(
   }
 }
 
+/** Scala wrapper for [[com.google.api.services.bigquery.model.Clustering]]. */
+case class Clustering(
```

Comment:
Probably worth to be `final`. I see this is also missing from the `TimePartitioning` where you copied the design.